app/models/package_manager/maven.rb

```diff
@@ -145,6 +147,8 @@ def self.versions(raw_project, name)
     end
 
     def self.retrieve_versions(versions, name)
+      # TODO: This should also look for the parent POM and if it exists + the child POM
```

Comment:
Wondering if it's worth logging the number of version packages with parents so we get a sense of what that volume looks like / if we can pick off any additional missing licenses with this type of fix.

Reply:
I think if there's a quick way to log something, go for it! Maybe something along the lines of 
`Rails.logger.info("[POM Parent Found] name=#{name} version=#{version}")`